tests/test-mongoc-collection.c

```diff
@@ -648,13 +697,22 @@ test_index_storage (void)
    bson_error_t error;
    bool r;
    bson_t keys;
-
-   mongoc_index_opt_init (&opt);
-   mongoc_index_opt_wt_init (&wt_opt);
+   const char *engine;
 
    client = mongoc_client_new (gTestUri);
    ASSERT (client);
 
+   /* Skip unless we are on WT */
+   engine = storage_engine(client);
+   if (strcmp(engine, "wiredtiger") != 0 &&
```

Comment:
I believe it's more idiomatic to say `strcmp(engine, "wiredtiger") &&  strcmp(engine, ...)`

"wiredTiger" will be the only one in the future, not sure if all lower case is still a possibility (or ever was)? Can you confirm that it is?